configs/mender_convert_config

```diff
@@ -102,6 +102,16 @@ MENDER_STORAGE_URL="https://d1b0l86ne08fsf.cloudfront.net"
 # Mender GitHub organization URL prefix
 MENDER_GITHUB_ORG="https://github.com/mendersoftware"
 
+# Whether the first partition of the input image should be ignored.
+#
+# For EFI targets, mender-convert assumes that the first partition of the input
+# image should be used as a base for the boot partition. If it does not exist,
+# mender-convert will create one instead
+#
+# Setting this variable to 1 will force ignoring this partition, instead using
+# the auto-generated one.
```

Comment:
Suggestion:

Setting this variable to 1 will force mender-convert to generate a boot part instead of using the first partition of the input image